[CI] watcher/usage/10_basic/Test watcher usage stats output fails

`watcher/usage/10_basic/Test watcher usage stats output` fails with an unexpected `watch_count_active`.

```
> Caused by: java.lang.AssertionError: field [watcher.count.active] is not greater than [$watch_count_active]
> Expected: a value greater than <0>
>      but: <0> was equal to <0>
>    at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
```

I can reproduce this locally with this command:

```
./gradlew :x-pack:qa:smoke-test-watcher-with-security:integTestRunner \
  -Dtests.seed=E3C29FF2DA312869 \
  -Dtests.class=org.elasticsearch.smoketest.SmokeTestWatcherWithSecurityClientYamlTestSuiteIT \
  -Dtests.method="test {yaml=watcher/usage/10_basic/Test watcher usage stats output}" \
  -Dtests.security.manager=true \
  -Dtests.locale=ru \
  -Dtests.timezone=America/Belize \
  -Dcompiler.java=10 \
  -Druntime.java=8
```

CI instances: 
- https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+periodic/6956/console
- https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+intake/2371/console

--- a/x-pack/plugin/src/test/resources/rest-api-spec/test/watcher/usage/10_basic.yml
+++ b/x-pack/plugin/src/test/resources/rest-api-spec/test/watcher/usage/10_basic.yml
@@ -1,8 +1,5 @@
 ---
 "Test watcher usage stats output":
-  - skip:
-      version: "all"
-      reason: AwaitsFix at https://github.com/elastic/elasticsearch/issues/33326
   - do:
       catch: missing
       xpack.watcher.delete_watch:

